src/Console/ConfigurationResolver.php

```diff
@@ -577,8 +577,8 @@ private function computeConfigFiles()
 
         if ($this->isStdIn() || 0 === \count($path)) {
             $configDir = $this->cwd;
-        } elseif (1 < \count($path)) {
-            throw new InvalidConfigurationException('For multiple paths config parameter is required.');
+        } elseif (1 < \count($path) && null === $this->options['rules']) {
+            throw new InvalidConfigurationException('For multiple paths the config or rules parameter is required.');
```

Comment:
so now, when `--rules` were passed, config will be taken from `$path[0]` implicitly? this is not good, if provided multiple paths for fixing will be different projects.

`/my/user/profile: $ php-cs-fixer fix projectA projectB`, now, `projectB` will be fixed using configuration of `projectA`
requiring an config file explicitly is removing that wrong usage of the tool, as person calling the tool knows that he is applying one config for multiple paths.

( @SpacePossum ref #4408 - using `defaultConfig` instead of one from `$path[]` is not better )